packages/inventory/src/Stocktake/DetailView/columns.ts

```diff
@@ -172,3 +172,24 @@ export const useStocktakeColumns = ({
     { sortBy, onChangeSortBy },
     [sortBy, onChangeSortBy]
   );
+
+export const useExpansionColumns = (): Column<StocktakeLine>[] =>
```

Comment:
In the non-expansionColumns.. the packSize needs to access data from `lines` rather than `rowData`.. currently the size only shows if non-grouped.

The expiry date isn't showing at all - needs to return a date rather than the formatted string

Reply:
Ah - sorry about that. Silly mistakes! Have made the changes and also fixed the pack size width in here also!